website/search/elastic_search.py

```diff
@@ -13,11 +13,10 @@
 from elasticsearch import (
     Elasticsearch,
     RequestError,
-    NotFoundError
+    NotFoundError,
+    ConnectionError
 )
 
-from requests.exceptions import ConnectionError
```

Comment:
It shouldn't be. I think this may have been a holdover from the last module we used (pyelasticsearch). I tested it manually, and it seems that the error thrown is definitely the elasticsearch one, whether it starts up without it or it goes down while the server is running.